### ISSUE TITLE
Online Mapping Note

### ISSUE THREAD

**Issue by mitchellryant** [TNRIS/dataHub#240]
Updated Service note to include direct connection to TNRIS ArcGIS Server. 

**Online Mapping Service**
This dataset is available as an online mapping service, as both as ArcGIS service and an OGC WMS service.  Use the URL below to connect to the OGC WMS service in your software. To connect to the TNRIS ArcGIS Server please use this url in our ESRI software, https://webservices.tnris.org/arcgis/services.
To use the service in your software package, consult the software help/support information.

**Comment by jwhaney** [comment]
made some small changes and re-wording. what do you think @mitchellryant ??

__Online Mapping Service__
This dataset is available as an online mapping service. A OGC WMS service and a ArcGIS service are available. To connect to the WMS service in your software, please copy the unique url provided in the box below. To access the TNRIS ArcGIS Server, please use the following url in your ESRI software and select from the list of available services: __https://webservices.tnris.org/arcgis/services__.

*For questions on using the service in your software package, consult the software help/support information*.

**Comment by jwhaney** [comment]
you think its too wordy?

![image](https://user-images.githubusercontent.com/13525011/70454413-ff970200-1a6f-11ea-8658-38180f470913.png)

**Comment by mitchellryant** [comment]
@jwhaney it look and reads fine to me.

**Comment by jwhaney** [comment]
ok cool. just created a pull request to get it merged into develop and be on the next deployment.

**Comment by jwhaney** [comment]
closed via PR #241
